README.md

```diff
@@ -1,2 +1,60 @@
+
+A key example of this is [pragmas](https://github.com/OpenDreamProject/OpenDream/wiki/Pragmas---Error-Emissions). Pragmas provide configurable compiler emissions for things such as runtime errors that *can* be detected at compiletime but BYOND fails to detect. Utilizing pragmas while maintaining BYOND compatibility is as simple as declaring them in their own code file and putting this in your code somewhere:
+```
+#ifdef OPENDREAM
+// Don't include the file of pragmas unless `OPENDREAM` is defined (which OpenDream defines for you)
+#include("/path/to/pragma/file.dm")
+#endif
+```
+
+Furthermore, OpenDream is intended to provide a solution for *everyone* and not any specific codebase. To prevent any one person from turning the project into their own new programming language, this RFC process exists to solicit input from the wider community on any major changes that would impact their current or future usage of OpenDream.
+
+## RFC Considerations
+
+Whenever new syntax is introduced, we need to ask:
+
+- Is it BYOND compatible?
+- Is it easy for machines and humans to parse?
+- Does it create grammar ambiguities for current and future syntax?
+- Is it stylistically coherent with the rest of the language?
+- Does it present challenges with editor integration like autocomplete?
+
+For changes in semantics, we should be asking:
+
+- Is behavior easy to understand and non-surprising?
+- Can it be implemented performantly today?
+- Is it compatible with type checking and other forms of static analysis?
+
+As the OpenDream userbase grows, reversing these decisions will only become more costly and have backwards compatibility implications.
+
+## RFC Process
+
+To open an RFC, a Pull Request must be opened which creates a new Markdown file in `docs/` folder. The RFCs should follow the template `TEMPLATE.md`, and should have a file name that matches the RFC's title.
+
+### RFC Drafts
+If you are in the early stages of designing the RFC and would like to solicit feedback on major considerations for your proposal, but it isn't ready for formal review, you can open your RFC as a Draft Pull Request. Once you are ready, mark it as ready for review and proceed to the below section:
+
+### RFC Review & Finalization
+Every open RFC will be open for at least two calendar weeks. This is to make sure that there is sufficient time to review the proposal and raise concerns or suggest improvements. The discussion points should be reflected on the PR comments; when discussion happens outside of the comment stream, the points salient to the RFC should be summarized as a followup.
+
+The first week should be used to resolve any major technical or design problems raised by the OpenDream team, though everyone is encouraged to provide input. After the first week and any major problems are resolved, a member of the OpenDream team should share the RFC in the following places to solicit feedback from the wider community for the second week:
```

Comment:
> ... raised by the OpenDream team

who is the OpenDream team? Maintainers on the repository?

Reply:
```suggestion
The first week should be used to resolve any major technical or design problems raised by the OpenDream team, though everyone is encouraged to provide input. After the first week and any major problems are resolved, a Maintainer from the OpenDream team should share the RFC in the following places to solicit feedback from the wider community for the second week:
```